docs/GettingStarted.md

```diff
@@ -29,8 +29,8 @@ We are going to work through creating a profile to generate random personal data
 
 Profiles are JSON files that describe the data you want to generate. They are composed of:
 
--   **fields** - an array of uniquely named fields (or columns).
--   **constraints** - an array of restrictions on the types and ranges of data permitted for the given column.
+-   **fields** - an array of uniquely named fields.
+-   **constraints** - an array of restrictions on the types and ranges of data permitted for a given field.
```

Comment:
I wonder if this should be "for given fields" - as it is, does it imply you can only one constrain one field in a profile?  I'm not entirely sure.

Reply:
Yes, I was unsure about this too for that reason. Perhaps "an array of restrictions on the types and ranges of data permitted for each field"? Although that could be taken as you have to specifiy a constraint for every field.